presto-hive/src/test/java/com/facebook/presto/hive/TestDomainTranslator.java

```diff
@@ -91,7 +99,7 @@ public void setup()
         columnExtractor = new SubfieldExtractor(
                 new FunctionResolution(metadata.getFunctionManager()),
                 (rowExpression, level, session) -> rowExpression,
-                TestingSession.SESSION).toColumnExtractor();
+                new TestConnectorSession()).toColumnExtractor();
```

Comment:
Instead of creating a new session class, just use
```java
TestingConnectorSession session = new TestingConnectorSession(new HiveSessionProperties(
        new HiveClientConfig().setNestedColumnsFilterEnabled(true),
        new OrcFileWriterConfig(),
        new ParquetFileWriterConfig())
        .getSessionProperties());